### PR TITLE
NumeRe.NumeRe version 1.1.3rc

### PR DESCRIPTION
**Pull request by numeredev**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21968)

### CODE REVIEW

File: manifests/n/NumeRe/NumeRe/1.1.3rc/NumeRe.NumeRe.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: NumeRe.NumeRe
+PackageVersion: 1.1.3rc
```
Comment (ItzLevvie) [inline manifests/n/NumeRe/NumeRe/1.1.3rc/NumeRe.NumeRe.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.1.3
```

File: manifests/n/NumeRe/NumeRe/1.1.3rc/NumeRe.NumeRe.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: NumeRe.NumeRe
+PackageVersion: 1.1.3rc
```
Comment (ItzLevvie) [inline manifests/n/NumeRe/NumeRe/1.1.3rc/NumeRe.NumeRe.yaml:5]:
```suggestion
PackageVersion: 1.1.3
```

File: manifests/n/NumeRe/NumeRe/1.1.3rc/NumeRe.NumeRe.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: NumeRe.NumeRe
+PackageVersion: 1.1.3rc
```
Comment (ItzLevvie) [inline manifests/n/NumeRe/NumeRe/1.1.3rc/NumeRe.NumeRe.installer.yaml:5]:
```suggestion
PackageVersion: 1.1.3
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Looks like Control Panel is showing this as `1.1.3` which means you'll have to update the version folder as well.

File: manifests/n/NumeRe/NumeRe/1.1.3rc/NumeRe.NumeRe.locale.en-US.yaml (hunk 1 above)
Comment (numeredev) [reply manifests/n/NumeRe/NumeRe/1.1.3rc/NumeRe.NumeRe.locale.en-US.yaml:5]: Hi @ItzLevvie!

You're talking about this version number?
grafik
That's an error in the installer. Thanks for mentioning it. Should also display 1.1.3rc

File: manifests/n/NumeRe/NumeRe/1.1.3rc/NumeRe.NumeRe.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: NumeRe.NumeRe
+PackageVersion: 1.1.3rc
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://sourceforge.net/projects/numere/files/Release%20Candidate/numere-win-v113rc.exe/download
+  InstallerSha256: 59A86C31075B5D1331F313D976D2415CA75D7C4EF40D068949B197EFDDB01EF8
```
Comment (numeredev) [inline manifests/n/NumeRe/NumeRe/1.1.3rc/NumeRe.NumeRe.installer.yaml:10]:
```suggestion
  InstallerSha256: ADE4429AF2FAD0175CE9804C43CF38533B53CD707DFA9D03BCC81429A2EF125F

```

File: manifests/n/NumeRe/NumeRe/1.1.3rc/NumeRe.NumeRe.locale.en-US.yaml (hunk 1 above)
Comment (numeredev) [reply manifests/n/NumeRe/NumeRe/1.1.3rc/NumeRe.NumeRe.locale.en-US.yaml:5]: I updated the installer and the SHA. Should now work as intended

**Comment by numeredev** [comment]
@ItzLevvie One question: will winget detect that 1.1.3 is newer than 1.1.3rc? How is a newer version determined?

File: manifests/n/NumeRe/NumeRe/1.1.3rc/NumeRe.NumeRe.installer.yaml (hunk 3 above)
Comment (numeredev) [reply manifests/n/NumeRe/NumeRe/1.1.3rc/NumeRe.NumeRe.installer.yaml:5]: Fixed in installer

File: manifests/n/NumeRe/NumeRe/1.1.3rc/NumeRe.NumeRe.yaml (hunk 2 above)
Comment (numeredev) [reply manifests/n/NumeRe/NumeRe/1.1.3rc/NumeRe.NumeRe.yaml:5]: Fixed in installer

**Review by numeredev — COMMENTED** [review]
Issue fixed in installer

**Review by OfficialEsco — APPROVED** [review]
image
In theory 1.1.3rc should be classified as the newer version depending on the sorting order, however the winget-cli/github sorting is weird, so you might want to keep an eye on this.

The schema also have a Channel variable, but its not implemented/supported as of now, which is why we for example use 
```
NumeRe.NumeRe                 # Stable version
NumeRe.NumeRe.Beta            # Beta Channel
NumeRe.NumeRe.Dev             # Dev Channel
NumeRe.NumeRe.Canary          # Canary Channel
NumeRe.NumeRe.RC              # Release Candidate in your case
```

**Pull request merged**